wicket/components-wicket/src/main/java/nl/rotterdam/nl_design/wicket/components/heading_1/README.md

```diff
@@ -0,0 +1,28 @@
+# Heading component voor Apache Wicket
+
+Gebruik de component als volgt:
+
+```java
+import nl.rotterdam.nl_design.wicket.components.heading_1.Heading1Behavior;
+
+public class MyPage extends BasePage {
```

Comment:
Deze code werkt helaas al niet meer  op feature/wicket-heading branch.

Mogelijk zijn we in staat om dit soort directories te genereren vanuit de 'echte' code-base. Is wel wat heavy enginering-werk,  maar ik zie wel mogelijkheden.